src/ui/DNA/pages/MintDnaFlow/MintDna.tsx

```diff
@@ -30,9 +30,9 @@ import { AddressDetails } from 'src/ui/pages/Receive/AddressDetails';
 import type { HTMLDialogElementInterface } from 'src/ui/ui-kit/ModalDialogs/HTMLDialogElementInterface';
 import { INTERNAL_ORIGIN } from 'src/background/constants';
 import { invariant } from 'src/shared/invariant';
-import { SidePanel } from 'src/ui/Onboarding/FAQ/SidePanel';
 import { useGasPrices } from 'src/ui/shared/requests/useGasPrices';
 import { useCurrency } from 'src/modules/currency/useCurrency';
+import { SidePanel } from 'src/ui/components/SidePanel';
```

Comment:
This is a bad generic name
Can we place it in `src/ui/Onboarding`? Or maybe in `src/ui/Onboarding/components`? It's onboarding-specific

Reply:
Moved to `src/ui/features/onboarding/shared` ✅